packages/create/templates/hello-prisma/src/operations.ts

```diff
@@ -1,8 +1,19 @@
-import { TransactionContext, Transaction, GetApi } from '@dbos-inc/dbos-sdk';
+import { HandlerContext, TransactionContext, Transaction, GetApi } from '@dbos-inc/dbos-sdk';
 import { PrismaClient } from "@prisma/client";
 
 export class Hello {
 
+  @GetApi('/') // Serve a quick readme for the app
+  static async readme(ctxt: HandlerContext) {
+    return '<html><body><p>' +
+           'Welcome to the DBOS Hello App!<br><br>' +
+           'Visit the route /greeting/:name to be greeted!<br>' +
+           'For example, visit <a href="/greeting/dbos">/greeting/dbos</a>.<br>' +
+           'The counter increments with each page visit.<br>' +
+           'If you visit with a new name like <a href="/greeting/alice">/greeting/alice</a> the counter starts at 1.' +
+           '</p></body></html>'
+  }
```

Comment:
When I ran `npm run lint` the linter complained:
```
  13:3   error  Static async method 'readme' has no 'await' expression                  @typescript-eslint/require-await
  13:23  error  'ctxt' is defined but never used. Allowed unused args must match /^_/u  @typescript-eslint/no-unused-vars
  20:32  error  Missing semicolon                                                       semi

✖ 3 problems (3 errors, 0 warnings)
```

So here's the suggested fix:

```suggestion
 static async readme(_ctxt: HandlerContext) {
    const text = `<html><body><p>
           Welcome to the DBOS Hello App!<br><br>
           Visit the route /greeting/:name to be greeted!<br>
           For example, visit <a href="/greeting/dbos">/greeting/dbos</a>.<br>
           The counter increments with each page visit.<br>
           If you visit with a new name like <a href="/greeting/alice">/greeting/alice</a> the counter starts at 1.
           </p></body></html>`;
    return Promise.resolve(text);
  }
```

Reply:
thanks @qianl15 - when you get a chance can you plz tell me exactly what you ran? And from what dir?

I want to be able to run it myself next time, but I can't repro the error message you got.

Reply:
Run `npm run lint` from `packages/create/templates/hello`. I'm getting the same issue.

Reply:
Code change should be in place now